ui/components/app/multichain-transaction-details-modal/multichain-transaction-details-modal.tsx

```diff
@@ -252,39 +272,26 @@ export function MultichainTransactionDetailsModal({
             gap={4}
           >
             {/* From */}
-            {transaction.type === TransactionType.Send
+            {type === TransactionType.Send
               ? accountComponent(t('from'), userAddress)
-              : assetInputs.map((input) =>
-                  accountComponent(t('from'), input.address),
-                )}
+              : accountComponent(t('from'), from?.address)}
```

Comment:
Summoning @darioAnongba here, but I don't think that change is valid. IIRC we were using `assetInputs` here because for Bitcoin, you could be using multiple UTXOs and you might want to log all the addresses they are coming from (I could be totally wrong, but I prefer to double-check before approving this change 😅)

Reply:
Sounds good, I wasn't aware of that decision, thanks for sharing, then I'm ok with this change! 😄

Reply:
We are currently aggregating by asset (that's why it's `assetInputs`, `assetOutputs`). For Bitcoin, we expect only 1 asset (BTC) and for Solana we don't allow multi-asset 

Edge cases where this logic will fail:
- Receving an asset on multiple of your addresses in the same transaction
- Receiving multiple assets in the same transaction
- Sending multiple assets in the same transaction

So the previous logic is better but the new one accommodates the current design and those edge cases will most likely never happen.